bitsandbytes/__main__.py

```diff
@@ -46,26 +32,31 @@ def generate_bug_report_information():
     print('')
 
     if 'CONDA_PREFIX' in os.environ:
-        paths = find_file_recursive(os.environ['CONDA_PREFIX'], '*cuda*so')
+        paths = find_file_recursive(os.environ['CONDA_PREFIX'], '*cuda*')
```

Comment:
We need to make sure that this does not match non-binary files. How can we make sure that this is correct in most environments?

Reply:
`dll`, `so`, or `dylib`  ext part appended later in the `find_file_recursive()`